content/docs/0.15.x/operate/install/index.md

```diff
@@ -71,20 +71,20 @@ Run your Keptn installation for free on GKE! If you [sign up for a Google Cloud
 
     - Set up the required permissions for your user:
 
-      ```console
+      ```
```

Comment:
please replace `console` with `shell` instead of removing it wherever applicable